scripts/smoke.sh

```
@@ -0,0 +1,278 @@
+  done
+fi
+
+docker ps --format 'table {{.Names}}\t{{.Image}}\t{{.Status}}\t{{.Ports}}' | grep -E "(^NAMES|${WEB_CONT}|${ADMIN_CONT}|${SPACE_CONT}|${LIVE_CONT})" || true
+
+echo
+echo "URLs:"
+$(printf '  %s\n' "${SELECTED[@]}" | awk '
+  /web/   {print "  web:   '"${web_url}"'"}
+  /admin/ {print "  admin: '"${admin_url}"'"}
+  /space/ {print "  space: '"${space_url}"'"}
+  /live/  {print "  live:  '"${live_url}"'"}')
```

Shellcheck (0.10.0)

[warning] Remove surrounding $() to avoid executing output (or use eval if intentional).

(SC2091)